src/redux/reducers/registration/__test__/registration.spec.js

```diff
@@ -0,0 +1,46 @@
+import { fromJS } from 'immutable';
+import reducer from '..';
+
+const CREATE_ACCOUNT_LOADING = 'CREATE_ACCOUNT_LOADING';
+const CREATE_ACCOUNT_SUCCESS = 'CREATE_ACCOUNT_SUCCESS';
+const SHOW_ERROR_MESSAGE = 'SHOW_ERROR_MESSAGE';
+
+describe('registration reducer', () => {
+    describe(`when ${CREATE_ACCOUNT_LOADING} action`, () => {
+        it('should set isLoading to true and empty the errorMessage', () => {
```

Comment:
fix this name because only is set isLoading to true the errorMessage needs to be removed